### PR TITLE
Use jll for Julia 1.3 and higher

### PR DESCRIPTION
**Pull request by odow**
x-ref: https://github.com/JuliaPackaging/Yggdrasil/issues/509#issuecomment-605516668

### CODE REVIEW

**Comment by kalmarek** [comment]
does `jll` artifact allow you to switch the underlying libraries to your own?

**Comment by odow** [comment]
> does jll artifact allow you to switch the underlying libraries to your own?

Not that I'm aware of. I guess we need an environment variable switch somewhere. Thoughts? I guess you're the one who knows the most about SCS.

**Comment by kalmarek** [comment]
yes, but I know next to nothing about artifacts ;-)
I see no changes to `build.jl`, so I believe it will ignore build file and check
https://github.com/JuliaOpt/SCS.jl/blob/bfc756ea4498c89c263f14cdb02dcadd063d17fa/deps/build.jl#L63
therein. Is there any way to bring back the option?

File: src/SCS.jl [hunk 1]
```diff
@@ -1,17 +1,23 @@
 module SCS
 
-if isfile(joinpath(dirname(@__FILE__), "..", "deps", "deps.jl"))
-    include(joinpath(dirname(@__FILE__), "..", "deps", "deps.jl"))
-else
-    error("SCS not properly installed. Please run Pkg.build(\"SCS\") and restart julia")
-end
+if haskey(ENV, "JULIA_SCS_LIBRARY_PATH") || VERSION < v"1.3"
```
Comment (odow) [inline src/SCS.jl]: @kalmarek will this work?

**Comment by kalmarek** [comment]
so even when `SCS.jl` depends on the artifact (i.e. `julia ^1.3`), when I call `Pkg.build("SCS")` the `deps/build.jl` will be run?

**Comment by odow** [comment]
I think so. Could you try it out?

**Comment by kalmarek** [comment]
works, thanks!

**Comment by odow** [comment]
@kalmarek I just rebased on top of your GPU commits. Assuming CI passes, I'll leave it with you to merge :)

**Comment by kalmarek** [comment]
cool, thanks!

**Pull request merged**